model/shift_design_objective.py

```diff
@@ -0,0 +1,19 @@
+
+from gurobipy.gurobipy import GRB, quicksum
+
+
+class ShiftDesignObjective:
+    def __init__(self, model, var, weights, shifts, time_periods):
+        self.model = model
+        self.shifts = shifts
+        self.time_periods = time_periods
+
+        self.add_objective(weights, var.y, var.delta)
+
+    def add_objective(self, weights, y, delta):
+        self.model.setObjective(
+            quicksum(y[t_marked, v] for t_marked, v in self.shifts) +
```

Comment:
Er det en grunn til at du ikke har vekter på `y`? Jeg tenker umiddelbart at det kan være lettere å justere modellen om man har vekter på hvert ledd, men kan godt hende det er unødvendig eller du har en god grunn for å la være